src/style.js

```diff
@@ -1,24 +1,12 @@
 import { Component } from 'react'
-import StyleSheetRegistry from './stylesheet-registry'
-
-const styleSheetRegistry = new StyleSheetRegistry()
+import RegistryContext from './stylesheet-registry-context'
```

Comment:
To simplify things a bit, we could create the `Provider` and `Consumer` in this file and then export the `Provider`.